styles.css

```diff
@@ -0,0 +1,88 @@
+body {
+  /*https://www.w3schools.com/css/css_rwd_images.asp*/
+  background-image: url(https://images5.alphacoders.com/443/443566.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  /*animation-name: example;
+  animation-duration: 20s;
+  animation-iteration-count: infinite;*/
+  height: 100%;
+  font-family: cursive;
+}
+
+@keyframes example {
+  0% {
+    background-color: blueviolet;
+  }
+  50% {
+    background-color: yellow;
+  }
+  100% {
+    background-color: blueviolet;
+  }
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  margin-top: auto;
+  width: 50%;
+  text-align: center;
+}
+
+header {
+  display: flex;
+  position: fixed;
+  width: 50%;
+  justify-content: center;
+}
+
+.login-modal {
+  margin-top: 30%;
+  width: 50%;
+  margin: 30% auto;
+  justify-content: space-evenly;
+  background-color: rgb(0, 140, 69, 0.5);
+}
+
+.login-modal-row {
```

Comment:
u ovoj klasi imas konflikt stilova pisanje CSS-a

Imas flexbox koji zahtjeva "povjerenje" u njega, znaci definira se flexbox u parentu a djeca se ne modificiraju puno jer o tome brine flexbox. Pod ne modicifiraju puno misli se na automatske margine, full width, sl. jer oni onemoce flexbox. Npr. postoje tri elementa i sva tri su preko cijele sirine (width + auto margin), to flexbox ne moze sloziti jer ce oni uvijek zauzimati cijelu sirinu i sve je isto kao da flexboxa i nema. Flexbox nacin bi bio postaviti flex, poravnati npr. horizontalno i vertikalno u sredinu, a djecu postaviti na npr width: 200px, bez horizontalnih margina  i onda ce flex znati posloziti content u sredinu.

Generalno pravilo je uzeti flexbox ili auto margine, kombinacije postoje ali u specificnim slucajevima (na razini jednog djeteta najcesce)